README.md

```diff
@@ -8,3 +8,13 @@ https://kudacam.netlify.app/
 
 Maintainer 
 [Vighnesh Manjrekar](https://github.com/VighneshManjrekar)
+
+## How to Contribute
```

Comment:
How to contribute section should not be generalized but about the contribution to current repo 
Also include commands and links for "fork" and "issues"
Use proper markdown to fix the formatting